src/main/java/Calculater.java

```diff
@@ -0,0 +1,166 @@
+import java.util.ArrayDeque;
+import java.util.Deque;
+
+public class Calculater {
+    private Deque<String> stack = new ArrayDeque<>();
+    private Deque<String> nstack = new ArrayDeque<>();
+    private int prev = 0;
+
+    public int calculate(String s) {
+        char[] modequation = s.toCharArray();
+        equationtostack(modequation);
+        while (!nstack.isEmpty())
+            if (nstack.peekLast() == "sub") {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use equals() to compare strings instead of '==' or '!='](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4535407)